WooCommerce/Classes/ViewRelated/Dashboard/StoreStats/StorePerformanceViewModel.swift

```diff
@@ -104,7 +104,6 @@ final class StorePerformanceViewModel: ObservableObject {
         }
         saveLastTimeRange(timeRange)
         shouldHighlightStats = false
-        usageTracksEventEmitter.interacted()
```

Comment:
This is removed because it's already tracked from the `StorePerformanceView` when selecting any item on the `StatsTimeRangePicker`.